src/sagemaker/jumpstart/estimator.py

```diff
@@ -0,0 +1,254 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module stores JumpStart implementation of Estimator class."""
+from __future__ import absolute_import
+
+
+from typing import Dict, List, Optional, Union
+from sagemaker import session
+from sagemaker.async_inference.async_inference_config import AsyncInferenceConfig
+from sagemaker.base_deserializers import BaseDeserializer
+from sagemaker.base_serializers import BaseSerializer
+from sagemaker.debugger.debugger import DebuggerHookConfig, RuleBase, TensorBoardOutputConfig
+from sagemaker.debugger.profiler_config import ProfilerConfig
+
+from sagemaker.estimator import Estimator
+from sagemaker.explainer.explainer_config import ExplainerConfig
+from sagemaker.inputs import FileSystemInput, TrainingInput
+from sagemaker.instance_group import InstanceGroup
+
+from sagemaker.jumpstart.factory.estimator import get_deploy_kwargs, get_fit_kwargs, get_init_kwargs
+from sagemaker.model_monitor.data_capture_config import DataCaptureConfig
+
+
+from sagemaker.serverless.serverless_inference_config import ServerlessInferenceConfig
+from sagemaker.workflow.entities import PipelineVariable
+
+
+class JumpStartEstimator(Estimator):
+    """JumpStartEstimator class.
+
+    This class sets defaults based on the model id and version.
+    """
+
+    def __init__(
+        self,
+        model_id: str,
```

Comment:
We could consider exposing another simpler "constructor" using `@classmethod`. Something like:

```
class JumpStartEstimator(Estimator):

    def simple_create(model_id: str, model_version: str, **kwargs):
          return cls(model_id=model_id, model_version=model_version, **kwargs)
```